pyro/params/param_store.py

```diff
@@ -170,15 +170,20 @@ def save(self, filename):
         with open(filename, "wb") as output_file:
             torch.save(self.get_state(), output_file)
 
-    def load(self, filename):
+    def load(self, filename, load_to_cpu=False):
         """
         Loads parameters from disk
 
         :param filename: file name to load from
         :type name: str
+        :param load_to_cpu: flag to load CUDA-trained parameters to CPU
+        :type name: bool
```

Comment:
this is actually a typo above it should be  `:type filename: str` and `:type load_to_cpu: bool`